api/app.py

```diff
@@ -29,13 +30,20 @@
 from core.util.cache import CachedData
 from scripts import InstanceInitializationScript
 
+
+def get_locale():
+    """The localization selection function to be used with flask-babel"""
+    languages = Configuration.localization_languages()
+    return request.accept_languages.best_match(languages, "en")
```

Comment:
The new implementation also requires us to use a default since None is no longer an acceptable return type. I chose "en".